src/axom/primal/geometry/Polyhedron.hpp

```diff
@@ -784,7 +791,7 @@ class Polyhedron
     // Expanded swap operations for HIP workaround.
```

Comment:
Is this workaround still necessary?

It's a bit weird that we need it for the `Octahedron` case but not `Hexahedron` or `Tetrahedron` cases. 
This might be hiding a larger problem (?)